docker/provision/docker-entrypoint.sh

```diff
@@ -51,6 +52,9 @@ case "$ARG" in
     upgrade)
         "${RALPH_LOCAL_DIR}/init-ralph.sh" upgrade
         ;;
+    ralphctl)
```

Comment:
Isn't it goint to do the same thing as 

docker run --entrypoint ralphctl <container> <extra_args>

?

Reply:
Yes, but before that it will load config from the config files.

Reply:
The line above will also read configuration files.

Reply:
I need the stuff that happens from line 40 of this file to have db connection configured for instance.

Reply:
ralphctl does that : https://github.com/allegro/ralph/blob/63316f4d9938c42cebe88ebdc1210a829b4b1dae/contrib/common/usr/bin/ralphctl#L5

Reply:
Ok, then this is uselss.